pages/index.js

```diff
@@ -1,11 +1,14 @@
 import Heading from "../components/Heading";
 import SubHeading from "../components/SubHeading";
```

Comment:
When you don't use this component, like i mentioned it below, you should also delete the import here.